ConsumerApi.Tests.Integration/Challenges/StepDefinitions/ChallengesApiStepDefinitions.cs

```diff
@@ -0,0 +1,90 @@
+using ConsumerApi.Tests.Integration.API;
+using ConsumerApi.Tests.Integration.Models;
+using ConsumerApi.Tests.Integration.Utils.Models;
+using ConsumerApi.Tests.Integration.Utils.StepDefinitions;
+using Microsoft.Extensions.Options;
+using TechTalk.SpecFlow.Assist;
+using static ConsumerApi.Tests.Integration.Utils.Configuration.Settings;
+
+namespace ConsumerApi.Tests.Integration.StepDefinitions;
+
+[Binding]
+[Scope(Feature = "Create a Challenge")]
+[Scope(Feature = "Get Challenge By Id")]
+public class ChallengesApiStepDefinitions : BaseStepDefinitions<Challenge>
+{
+    private readonly ChallengesApi _challengeApi;
+    private string _challengeId;
+
+    public ChallengesApiStepDefinitions(IOptions<HttpConfiguration> httpConfiguration, ChallengesApi challengeApi) : base(httpConfiguration, new HttpResponse<Challenge>())
+    {
+        _challengeApi = challengeApi;
+        _challengeId = string.Empty;
+    }
+
+    [Given(@"a Challenge c")]
+    public async Task GivenAChallengeC()
+    {
+        var challengeResponse = await _challengeApi.CreateChallenge(_requestConfiguration);
+        challengeResponse.IsSuccessStatusCode.Should().BeTrue();
+
+        _challengeId = challengeResponse.Content!.Result!.Id;
+        _challengeId.Should().NotBeNullOrEmpty(because: "Required value for 'Id' is missing.");
```

Comment:
This is the only line where a `because` is defined. You should remove it to stay consistent